appidmanagementv4/app_id_management_v4.go

```diff
@@ -1372,9 +1424,18 @@ func (appIdManagement *AppIDManagementV4) StartSignUpWithContext(ctx context.Con
 	if startSignUpOptions.Active != nil {
 		body["active"] = startSignUpOptions.Active
 	}
+	if startSignUpOptions.LockedUntil != nil {
+		body["lockedUntil"] = startSignUpOptions.LockedUntil
+	}
+	if startSignUpOptions.DisplayName != nil {
+		body["displayName"] = startSignUpOptions.DisplayName
+	}
 	if startSignUpOptions.UserName != nil {
 		body["userName"] = startSignUpOptions.UserName
 	}
+	if startSignUpOptions.Status != nil {
+		body["status"] = startSignUpOptions.Status
```

Comment:
can we validate the status value before submitting? Accepted values are `PENDING` and `CONFIRMED`. We had some unexpected behavior in the past when users were not setting a correct value, we are defaulting to PENDING now, but still good to be explicit

Reply:
actually we would need a validation only in the setter, I left a comment there as well

Reply:
I'll see if I can do that with an update to openapi spec, so far this code was fully generated would be nice to keep it that way

Reply:
since OpenAPI doesn't natively support enums, I'll resolve this

Reply:
ok, for reference: https://github.ibm.com/arf/planning-sdk-squad/issues/1439 that feature is not yet implemented on go sdk generator